contracts/IndexGovernance.sol

```diff
@@ -53,7 +59,17 @@ contract IndexGovernance is Maintenance {
         }
         require(totalWeights == 10000, "IndexGovernance: TOTAL_WEIGHTS");
 
-        proposal = Proposal(++lastProposalId, _assets, _weights, block.number.add(_duration), msg.sender, 0, 0);
+        proposal = Proposal(
+            ++lastProposalId,
+            _assets, _weights,
+            block.number.add(_duration),
+            msg.sender,
+            0,
+            0,
+            _title,
+            _description,
+            _link
+        );
         emit ProposalCreated(_assets, _weights, _duration, msg.sender);
```

Comment:
Emitted event must also include these fields. Otherwise, serve development would not be able to properly show the  history of voting.